README.md

```diff
@@ -77,6 +77,24 @@ for a complete example.
 
 See [CONTRIBUTING.md](CONTRIBUTING.md)
 
+We hold regular meetings. See details at [community page](https://github.com/open-telemetry/community#python-sdk).
```

Comment:
Here's what I would write here to replace the content on the community page:

----

We meet weekly on Thursday at 8AM PT. The meeting is subject to change depending on contributors' availability. Check the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com) for specific dates.

Meetings take place via Zoom video conference: https://zoom.us/j/6729396170.

Meeting notes are available as a public [Google doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit). For edit access, get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-python).